getting-started/dcos/linkerd-package-options.json

```diff
@@ -0,0 +1,13 @@
+{
+  "linkerd": {
+    "admin-port": 9990,
+    "config-filename": "linkerd.yaml",
+    "config-uri": "https://raw.githubusercontent.com/BuoyantIO/linkerd-examples/master/getting-started/dcos",
+    "cpus": 0.25,
+    "instances": 1,
+    "mem": 256.0,
+    "name": "linkerd-example",
+    "routing-port": 4140,
+    "resource-role": "slave_public"
```

Comment:
unfortunately `resource-role` is only supported in the namerd package. the intent was that we want folks running a linkerd on every node. to ensure you get a linkerd onto the public node, you need to set `instances` to the total number of nodes.

we also could consider supporting `resource-role` for the linkerd universe package.